server/views/pages/prisons/incentiveGroups/addIncentiveGroup.njk

```diff
@@ -0,0 +1,93 @@
+{% extends "partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions %}
+{%- from "govuk/components/button/macro.njk" import govukButton -%}
+{%- from "govuk/components/input/macro.njk" import govukInput -%}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set pageHeaderTitle = 'Add a incentive group' %}
+{% set pageTitle = applicationName + " - Incentive groups - " + pageHeaderTitle %}
+
+{% set primaryNavItems = [
+  {
+    text: "Home",
+    href: "/"
+  },
+  {
+    text: "Prisons",
+    href: "/prisons",
+    active: true
+  }
+] %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ prison.name }}</span>
+        {{ pageHeaderTitle }}
+      </h1>
+
+    </div>
+  </div>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      {% include "partials/errorSummary.njk" %}
+      {# Build button items for session template range filter #}
+
+      {% set incentiveLevelItems = [] %}
+      {% for rangeKey, description in incentiveLevels %}
+        {% set incentiveLevelItems = (incentiveLevelItems.push({
+          value: rangeKey,
+          text: description
+        }), incentiveLevelItems)%}
+      {% endfor %}
+
+      <form action="/prisons/{{prison.code}}/incentive-groups/add" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+
+        {{ govukInput({
+          label: {
+            html: 'Incentive group name',
```

Comment:
```suggestion
            html: 'Incentive level group name',
```
(n.b. I'm assuming here that we agreed they could be `incentiveGroups` in code but text - like on  sub-nav tabs - would be `incentive level groups`. Ignore me if I'm wrong!)